README.md

```diff
@@ -13,52 +13,68 @@ in publications:
 
 * Olsson, Mats HM, Chresten R. Sondergaard, Michal Rostkowski, and Jan H. Jensen. "PROPKA3: consistent treatment of internal and surface residues in empirical pKa predictions." Journal of Chemical Theory and Computation 7, no. 2 (2011): 525-537. doi:[10.1021/ct100578z](https://doi.org/10.1021/ct100578z)
 
-See [propka.ki.ku.dk](http://propka.ki.ku.dk/) for the PROPKA web server,
```

Comment:
I would add a note on versioning. If you want to commit to [semantic versioning](https://semver.org/) then here would be a good place to say so.
```suggestion

### PROPKA versions

The code in this repository is named _PROPKA 3_ and it is based on the original PROPKA 3.1 release (described in the papers above). It has undergone various changes, which is reflected in the version numbering. For instance, version 3.2 contains a number of API changes and code refactoring that introduce incompatibilities between the original 3.1 release and the more recent versions. In the future, we will increase the minor version number to indicate further changes to the code base (e.g., release 3.3 or 3.4). The major release number is not expected to change unless major changes to the underlying algorithms are implemented.
```

Reply:
Implemented in 73e956f141930dd5ae366b7eeac8e1437f1c993f